mrobosub_sim/mrobosub_sim/hal.py

```diff
@@ -43,18 +46,23 @@ class Targets(Enum):
 class SensorData:
     depth: Float32
     dvl: Dvl
-    imu_ins: Imu_INS
```

Comment:
Please revert this. The Imu topic type does not contain all of the data produced and consumed by the sim and our imu sensor.

Reply:
There is an orientation field on the Imu message but I'm not publishing it because that isn't how the manufactures node does it

Reply:
The PIMU will be used for IEKF and INS is the data from an IMU onboard AHRS that updates at a faster rate than our InEKF ever will and also incorporates magnetometer updates. The INS also provides dt values which are not present in the ROS Imu message type. Please revert this change.